### PR TITLE
Add option to specify zone in compute_instance

### PR DESCRIPTION
**Pull request by MartinPetkov**
Also added a test that creates two instances and ensures they're in the same specified zone.

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_generate_modules
 The modules need to be regenerated. Please run `make_build`. 
 ```bash 
Checking submodule's files generation
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' /workspace/examples/compute_instance/simple/README.md /tmp/tmp.Vt4thb4wcb/workspace/examples/compute_instance/simple/README.md
16a17
> | zone | The GCP zone to create and test resources in | `string` | `null` | no |
Error: submodule's files generation has not been run, please run the
'make build' command and commit changes
 ``` 
- ⚠️check_documentation
 The documentation needs to be regenerated. Please run `make generate_docs`. 
 ```bash 
Checking for documentation generation
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' '--exclude=autogen' '--exclude=*.tfvars' /workspace/examples/compute_instance/simple/README.md /tmp/tmp.8ReXUo7dE0/generate_docs/workspace/examples/compute_instance/simple/README.md
16a17
> | zone | The GCP zone to create and test resources in | `string` | `null` | no |
Error: Documentation generation has not been run, please run the
'make docker_generate_docs' command and commit the above changes.
 ```

File: examples/compute_instance/simple/variables.tf [hunk 1]
```diff
@@ -27,6 +27,12 @@ variable "region" {
   default     = "us-central1"
 }
 
+variable "zone" {
+  description = "The GCP zone to create and test resources in"
```
Comment (bharathkkb) [inline examples/compute_instance/simple/variables.tf:31]:
```suggestion
  description = "The GCP zone to create resources in"
```

File: modules/compute_instance/variables.tf [hunk 2]
```diff
@@ -64,3 +64,8 @@ variable "region" {
   default     = null
 }
 
+variable "zone" {
+  type        = string
+  description = "Zone where the instances should be created. If not specified, will spread instances across available zones in the region."
```
Comment (bharathkkb) [inline modules/compute_instance/variables.tf:69]:
```suggestion
  description = "Zone where the instances should be created. If not specified, instances will be spread across available zones in the region."
```

File: test/fixtures/compute_instance/simple_zone/outputs.tf [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  value       = module.instance_simple.instances_self_links
+}
+
+output "project_id" {
+  description = "The GCP project to use for integration tests"
+  value       = var.project_id
+}
+
+
+
```
Comment (bharathkkb) [inline test/fixtures/compute_instance/simple_zone/outputs.tf:28]: nit: extra newlines

**Review by bharathkkb — DISMISSED** [review]
Thanks for the PR @MartinPetkov

**Comment by bharathkkb** [comment]
fixes https://github.com/terraform-google-modules/terraform-google-vm/issues/134

**Comment by MartinPetkov** [comment]
> fixes #134

Note that I didn't modify the umig module. Should I do that as well?

**Comment by bharathkkb** [comment]
@MartinPetkov feel free to add that if you have bandwidth, otherwise it's clear from the title it only added to compute instance

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**